src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+function ifSuccess(successMessage) {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = successMessage;
+  document.body.appendChild(div);
+}
```

Comment:
The ifSuccess function creates a new div element every time it is called. This could potentially lead to performance issues if the function is called frequently. Consider reusing existing elements or using a more efficient way to update the DOM.